CHANGELOG

```diff
@@ -1,3 +1,22 @@
+Release 117 Biskupin (2019-10-02)
```

Comment:
There are 16 commits here but 17 in the PR (not counting the `Prepare for release` commit). One is missing.